Removed backend route table definition, because it was inexistent

mmgari opened the pull request.

mmgari committed:
Removed backend route table definition, because it was inexistent

fwelschen commented on main.tf:
```diff
@@ -381,12 +381,3 @@ resource "aws_vpn_gateway_route_propagation" "public" {
   vpn_gateway_id = "${aws_vpn_gateway.this.id}"
```
En esta parte del codigo
```
 ###############################
 # VPN private route propagation
 ###############################
 resource "aws_vpn_gateway_route_propagation" "private" {
   count          = "${var.enable_vpn_route_private_propagation ? length(var.private_subnets) : 0}"
   vpn_gateway_id = "${aws_vpn_gateway.this.id}"
   route_table_id = "${element(aws_route_table.private.*.id, count.index)}"
```
Tendrias que iterar por el maximo numero de `private_subnets` y `backend_subnets`

mmgari committed:
Fix wrong number of subnets for route creation

I have to evaluate the maximum value between private and backend subnets, to create the propper number of private route tables

fwelschen reviewed (approved):
:shipit:
